mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+from .base import get_coefs, BaseEstimator, _check_estimator
+from ..utils import warn
+
+
+class ReceptiveField(BaseEstimator):
+    """Fit a receptive field model.
+
+    Parameters
+    ----------
+    lags : array, shape (n_lags,)
+        Time lags to be included in the model. This will be combined
+        with `sfreq` to convert these values into indices in the data.
+        If `sfreq` == 1, then these should be indices. If not, then
+        time lags should be in seconds.
+    feature_names : array, shape (n_features,)
+        Names for the input features to the model.
+    sfreq : float
+        The sampling frequency of the data.
+    model : instance of sklearn model | None
+        The model used in fitting inputs and outputs. This can be any
+        sklearn-style model that contains a fit and predict method.
```

Comment:
What does `None` mean? If it means `Ridge` then we should allow string args and default it to `'ridge'`